scripts/jenkins/qat/tc-deploy.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+#
+# Copyright 2019 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Deploys current jenkins-build test image for dpdk-app (crypto-perf)
+# in the cluster.
+
+set -o pipefail
+set -o xtrace
+set -o errexit
+
+REPO_ROOT=${WORKSPACE:-$(realpath $(dirname $0)/../../..)}
+TCNAME=${TCNAME:-crypto}
+TCNUM=${TCNUM:-1}
+kubectl apply -k ${REPO_ROOT}/deployments/qat_dpdk_app/test-${TCNAME}${TCNUM}/
+kubectl wait --for=condition=Initialized pod/qat-dpdk-test-${TACNAME}-perf-tc${TCNAME} --timeout=5m && sleep 60s
```

Comment:
wait for `Completed`?

Reply:
`kubectl wait` never reaches Completed and drops in the timeout set, this wait function seems to be in experimental phase yet and only supports the valid condition names listed in `type` (see first link below):

https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-conditions

https://github.com/kubernetes/kubernetes/issues/66439

I think keep this `Initialized` is ok by now, or should we move to something like `rollout status` (not quite sure if works for pods)?

Reply:
The current implementation has two problems: 1) it only gives 60s for the pod to complete (what if the test runs longer than 60s) 2) you don't check the test result or if the pod fails for some reason.

`kubectl describe pod ...` gives the conditions to wait. How about `Ready`?

Reply:
@mythi yes, Ready seems to be fine  using 'kubectl apply && kubectl wait ' together.